src/v2/Apps/Artist/ArtistApp.tsx

```diff
@@ -47,7 +47,7 @@ const ArtistApp: React.FC<ArtistAppProps> = ({ artist, children, match }) => {
 
   const showOverviewTab = hasOverviewContent(artist)
   const showArtworksTab = artist?.statuses?.artworks
-  const showAuctionLotsTab = artist?.statuses?.auctionLots
+  const showAuctionResultsTab = artist?.counts?.auctionResults
```

Comment:
`artist?.statuses?.auctionResults` doesn't seem to be reliable. We use `artist?.counts?.auctionResults` already for the [search auto-suggest quick navigation links](https://github.com/artsy/force/blob/master/src/v2/Components/Search/SearchBar.tsx#L356) in Force and Eigen.